flask_jwt_extended/config.py

```diff
@@ -87,7 +87,7 @@ def header_name(self):
 
     @property
     def header_type(self):
-        return current_app.config['JWT_HEADER_TYPE']
+        return current_app.config.get('JWT_HEADER_TYPE')
```

Comment:
Did this need to be changed? I think the `setdefault` in jwt_manager.py should guarantee that this is always present

Reply:
oh that was an accident, I didn't mean to change that, I had changed it to get to test for a bug, but then meant to change it back.